reference/core/ingress-controller.md

```diff
@@ -1,72 +1,18 @@
 # Ambassador as an Ingress Controller
 
-Starting with version 0.80.0, the Ambassador Edge Stack can act as a Kubernetes [ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/), reading configuration data from Kubernetes [`Ingress`](https://kubernetes.io/docs/concepts/services-networking/ingress/) resources. This makes it easier to work with other `Ingress`-oriented tools within the Kubernetes ecosystem, and it makes it easier for users migrating from other ingress controllers to try the Ambassador Edge Stack.
+An `Ingress` resource is a popular way to expose Kubernetes services to the Internet. In order to use `Ingress` resources, you need to install an [ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/). The Ambassador Edge Stack can function as a full-fledged Ingress controller, making it easy to work with other `Ingress`-oriented tools within the Kubernetes ecosystem.
```

Comment:
fully-fledged

Reply:
Hmmmm. I've always heard "full-fledged" (in the technical world anyway).

Reply:
Well, "fledge" is a verb, and "full[y]" is modifying that verb, making it an adverb not an adjective, and the adverbs usually end in "ly".

Reply:
Fixed here for EA, and in https://github.com/datawire/ambassador-docs/pull/186 for GA.